client/web/compose/src/store/ui.js

```diff
@@ -35,6 +38,14 @@ export default function (ComposeAPI) {
 
       previousPages: (state) => state.previousPages,
 
+      previousPage: (state) => {
+        if (state.previousPage && state.previousPage.name) {
```

Comment:
I would move this to the setPrevious page, so you dont need to check in the getter